Marsiple/App/Todos/TodosViewController.swift

```diff
@@ -40,13 +36,19 @@ extension TodosViewController: UITableViewDataSource {
         guard let cell = todosView.tableView.dequeueReusableCell(withIdentifier: "TodoTableViewCell", for: indexPath) as? TodoTableViewCell
             else { return UITableViewCell() }
         cell.updateProperties(withTodo: todos[indexPath.section][indexPath.row])
-        cell.delegate = self
-        cell.indexPath = indexPath
+        cell.didSelectCheckBox = didSelectCheckBox(atIndexPath: indexPath)
         return cell
     }
 }
 
 extension TodosViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let todoFormViewController = TodoFormViewController(todo: todos[indexPath.section][indexPath.row])
+        todoFormViewController.todoCompletedIsEdited = didSelectCheckBox(atIndexPath: indexPath)
+        todoFormViewController.todoTextIsEdited = didChangeText(atIndexPath: indexPath)
```

Comment:
Lets create a closure here with `[weak self]` and call the `didSelectCheckBox` method directly, without it returning a `closure`, but acting directly.

For both the `todoCompletedIsEdited` and `todoTextIsEdited`.